.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: test
+run-name: ${{ github.actor }} runs test
+on:
+  pull_request:
+    types: [opened]
+jobs:
+  check-bats-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.12.x'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install poetry
+          poetry install
+      - name: Run review
+        env:
+          GIT_API_TOKEN: ${{ secrets.GIT_API_TOKEN }}
+          GOOGLE_API_KEY: ${{ secrets.GOOGLE_API_KEY }}
+          PULL_REQUEST_NUMBER: ${{ github.event.pull_request.number }}
+          COMMIT_ID: ${{ github.event.pull_request.head.sha }}
+          PR_REPO_NAME: ${{ github.event.pull_request.head.repo.full_name }}
+          PR_BRANCH_NAME: ${{ github.event.pull_request.head.ref }}
```

Comment:
I suggest using the `google-api-python-client` library to interact with the GitHub API instead of directly using `requests`. This library provides a more convenient and robust way to handle authentication and API requests.